ops/storage-ddls/cassandra-setup.cql

```diff
@@ -79,87 +76,126 @@ CREATE TYPE features (
 );
 
 CREATE TABLE computedtiles (
+    tileid text,
     tilex int,
     tiley int,
     tilez int,
-    tileid text,
     periodstartdate timestamp,
     periodenddate timestamp,
     periodtype text,
     period text,
-    pipeline text,
-    sourceid text,
-    topic text,
-    topiclangcode text,
+    pipelinekey text,
+    externalsourceid text,
+    topicconjunctionid uuid,
     insertion_time timestamp,
     heatmap text,
     placeids frozen<set<text>>,
     computedfeatures frozen<features>,
-    PRIMARY KEY ((tilex, tiley, tilez), periodstartdate, periodenddate, pipeline, sourceid, topic)
+    PRIMARY KEY ((period, periodtype, pipelinekey, topicconjunctionid), externalsourceid, tilez, tilex, tiley, periodstartdate, periodenddate)
 );
 
-CREATE INDEX ON computedtiles (tileid);
-CREATE INDEX ON computedtiles (period);
-CREATE INDEX ON computedtiles (computedfeatures);
-CREATE INDEX ON computedtiles (periodtype);
+create table topicconjunctions(
+    topic text,
+    topicconjunctionid uuid,
+    conjunctivekeywordcombination map<text, int>,//map value is representative of the aggregated mention count displayed in the dashboard
+    PRIMARY KEY (topic, topicconjunctionid)
+);
+
+CREATE INDEX ON topicconjunctions ( KEYS (conjunctivekeywordcombination) );
```

Comment:
Is this to facilitate looking up the right id for a given set of keywords? This is going to be a common operation, so I wonder if it'd be easier for us to just rely on a convention like the conjunctionid is always the underscore-joined sorted set of the keywords (e.g. terms "isis, bomb" maps to id "bomb_isis") so that we can analytically determine the ID that we should query for without having to hit the database for the lookup.